packages/seedless-onboarding-controller/src/SeedlessOnboardingController.ts

```diff
@@ -487,6 +507,14 @@ export class SeedlessOnboardingController<EncryptionKey> extends BaseController<
           authPubKey: newAuthKeyPair.pk,
         });
         this.#resetPasswordOutdatedCache();
+
+        // store the keyring encryption key if it exists
+        if (keyringEncryptionKey) {
+          await this.#storeKeyringEncryptionKey(
```

Comment:
i think it would be cleaner if we call this function inside "updateVault" function since we are updating `encryptedSeedlessKey` in updateVault as well. 

Also `createNewVaultWithAuthData` in line 500 of this function already calls `updateVault`.

Reply:
Similarly `loadKeyringEncryptionKey` from this function can be moved to updateVault.